tools/archives/esap_integration/astronomical_archives.xml

```diff
@@ -0,0 +1,95 @@
+<tool id="astronomical_archives" name="Astronomical archives" version="1.0.0">
```

Comment:
I think we should remove this tool from this PR and keep it in a separate future PR.